id.html

```diff
@@ -0,0 +1,238 @@
+    </div>
+    <a id="down-arrow" href="#kali-highlights">
+      <div>
+      </div>
+    </a>
+  </header>
+  <section id="kali-highlights">
+    <div>
+      <div>
+        <h1>Gratis dan Canggih</h1>
+      </div>
+    </div>
+    <h3>Menghabiskan lebih sedikit energi dan waktu, membantu menyelesaikan pekerjaan mu, memperbaiki perngalaman game anda, dan tidak lagi menderita dengan akun yang burik.</h3>
+    <ul>
+      <li>
+        <a href="javascript:;">
+          <h4>
+            <i class="iconfont icon-youxiyouxisheji"
+              style="margin-right: 0.4em;line-height: 1;font-size: 25px;vertical-align: bottom;">
+            </i>Player
+          </h4>
+        </a>
+        <p>Mode serangan ganda, untuk menghindari terdeteksi oleh server dari damage yang tinggi. Hapus cooldown, ledakan elemen tidak perlu menunggu cooldown, karena variasi permainan, server tidak akan memeriksanya. Mode tak terkalahkan probabilitas yang sudah di sesuaikan.</p>
+      </li>
+      <li>
+        <a href="javascript:;">
+          <h4>
+            <i class="iconfont icon-dituleiditu"
+              style="margin-right: 0.4em;line-height: 1;font-size: 25px;vertical-align: bottom;">
+            </i>Dunia
+          </h4>
+        </a>
+        <p>Secara otomatis menyelesaikan tantangan dunia, memasak makanan secara otomatis, menghancurkan objek secara otomatis, memancing ikan secara otomatis, mengambil item secara otomatis, mengambil item dengan jangkuan yang jauh, fairy spirits secara otomatis, dialog NPC otomatis (dapat di percepat), menebang pohon secara otomatis, menyamarkan Waktu In-game, Cloing global, Kill Aura, Membuat makhluk adsorpsi, drop adsorpsi, dengan teleportasi otomatis agar tidak membuat tangan anda lelah, disesuaikan untuk hati anda.</p>
+      </li>
+      <li>
+        <a href="javascript:;">
+          <h4>
+            <i class="iconfont icon-24gl-paperPlane"
+              style="margin-right: 0.4em;line-height: 1;font-size: 25px;vertical-align: bottom;">
+            </i>Teleport
+          </h4>
+        </a>
+        <p>Teleportaasi sekali klik, teleportasi sekali klik dari peta, teleportasi khusus (dengan file json untun membuka kunci dunia baru), teleportasi sepenuhnya otomatis (delay yang dapat disesuaikan), teleportasi peta (titik tempat teleportasi mana), teleportasi titik quest  (perlu melacak quest).</p>
+      </li>
+      <li>
+        <a href="javascript:;">
+          <h4>
+            <i class="iconfont icon-toushifenxi"
+              style="margin-right: 0.4em;line-height: 1;font-size: 25px;vertical-align: bottom;">
+            </i>Perspektif
+          </h4>
+        </a>
+        <p>Mode Box drawing, Mode drawing Box model, Mode drawing empat border, Pelacak Linear, Pelacak Petunjuk, Draw rekan satu tim (kebanyakaan digunakan untuk petak umpet), Draw nama item, Draw darah monster, Memantau jarak jauh seacara real time, Custom color, Custom ukuran, Custom filter, Perspektif pada semua sumber daya umum.</p>
```

Comment:
change it to `Mode draw kotak, mode draw model kubus, mode draw garis empat sudut, pelacak linier, pelacak penunjuk, draw rekan setim (sering digunakan untuk bermain petak umpet atau Windtrace), draw nama item, draw HP monster, memonitor jarak secara real-time, kustom warna, kustom ukuran, kustom isian (fill), dapat melihat melalui semua sumber daya umum.`